peps/pep-0750.rst

```diff
@@ -809,79 +784,172 @@ At the time of this PEP's announcement, a fully-working implementation is
 
 This implementation is not final, as the PEP discussion will likely provide changes.
 
+
 Rejected Ideas
 ==============
 
+This PEP has been through several significant revisions. In addition, quite a few interesing
+ideas were considered both in revisions of :pep:`501` and in the `Discourse discussion <https://discuss.python.org/t/pep-750-tag-strings-for-writing-domain-specific-languages/60408/196>`_.
+
+We attempt to document the most significant ideas that were considered and rejected.
 
-Enable Exact Round-Tripping of ``conv`` and ``format_spec``
------------------------------------------------------------
 
-There are two limitations with respect to exactly round-tripping to the original
-source text.
+Arbitrary String Literal Prefixes
+---------------------------------
 
-First, the ``format_spec`` can be arbitrarily nested:
+Inspired by `JavaScript tagged template literals <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#tagged_templates>`_,
+an ealier version of this PEP allowed for arbitrary "tag" prefixes in front
+of literal strings:
 
 .. code-block:: python
 
-    mytag'{x:{a{b{c}}}}'
+    my_tag'Hello {name}'
+
+The prefix was a special callable called a "tag function" that received the 
+equivalent of this PEP's ``Template.args``, processed it, and returned an arbitrary
+value:
+
+.. code-block:: python
```

Comment:
Long sentence alert, I suggest splitting after "tag function". Also: "callable called" is awkward.